management_api_app/tests/test_db_repository/test_resource_spec_repo.py

```diff
@@ -0,0 +1,32 @@
+import db.repositories.resource_specs
+from unittest.mock import patch, MagicMock
+
+
+@patch('azure.cosmos.CosmosClient')
+def test_get_by_name_exisiting_document_returns_document(cosmos_client):
+    sut = db.repositories.resource_specs.ResourceSpecRepository(cosmos_client)
+    sut.container.query_items = MagicMock()
```

Comment:
Can we use the same mocking as we have used earlier? to keep the code base consistent (see the other tests)